setup.py

```diff
@@ -21,7 +21,10 @@ def run_tests(self):
 setup(
     name='vsutil',
     version='0.1.0',
-    py_modules=['vsutil'],
+    packages=find_packages(exclude=['tests']),
+    package_data={
+        'vsutil': ['py.typed']
+    },
     url='https://encode.moe/vsutil',
```

Comment:
This url should be fixed since it currently 404s.

Reply:
Let me fix that.

Reply:
Fixed.